wpiutil/src/main/native/include/wpi/Logger.h

```diff
@@ -80,7 +80,7 @@ class Logger {
 #define WPI_INFO(inst, format, ...) \
   WPI_LOG(inst, ::wpi::WPI_LOG_INFO, format __VA_OPT__(, ) __VA_ARGS__)
 
-#ifdef NDEBUG
+#if 0
```

Comment:
Maybe add a different macro that can be used to control logging? I suggest `NO_NT_LOG` (so it logs by default, unless the macro is defined).

On second thought, would macros defined by teams even affect this? If not, never mind.

Reply:
No, macros defined by teams wouldn't affect this (except where they use these macros themselves, which is likely pretty rare), as where we use this is in our compiled .cpp files.

Reply:
In that case, what's the point in the `#if 0`? Why only conditionally define?

Reply:
Yes, we can probably just remove the block since we're not intending to use it.